fonts/fonts.css

```diff
@@ -0,0 +1,24 @@
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('montserrat-bold-webfont.woff2') format('woff2'),
+       url('montserrat-bold-webfont.woff') format('woff');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('montserrat-medium-webfont.woff2') format('woff2'),
+       url('montserrat-medium-webfont.woff') format('woff');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO__Oxana/pullRequest?prid=5565108)